ote/ote/interfaces/classification/dataset.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
should not this file be in `ote/ote/tasks/classification/` ?

Reply:
Tasks are supposed to contain reference implementation of interfaces. At the same time the `interfaces` folder stores requirements to all supported usecases. This design allows to have several tasks that use `IClassificationDataset`.